test/bdd/docker.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package bdd
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// DockerHelper helper for docker specific functions
+type DockerHelper interface {
```

Comment:
Doesn't this also seem like a sub package ... like dockerutil?

Reply:
yes

Reply:
done